contracts/modules/STO/USDTiered/USDTieredSTO.sol

```diff
@@ -231,6 +244,12 @@ contract USDTieredSTO is USDTieredSTOStorage, STO {
             require(_ratePerTierDiscountPoly[i] <= _ratePerTier[i], "Invalid discount");
             tiers.push(Tier(_ratePerTier[i], _ratePerTierDiscountPoly[i], _tokensPerTierTotal[i], _tokensPerTierDiscountPoly[i], 0, 0));
         }
+        if (preMintAllowed) {
+            uint256 oldCap = securityToken.balanceOf(address(this));
+            uint256 newCap = _getTotalTokensCap();
+            if (oldCap < newCap)
+                securityToken.issue(address(this), (newCap - oldCap), "");
```

Comment:
Don't we need to burn (redeem) tokens if `oldCap > newCap` as well?

Reply:
Yes, this is a possibility It is an open question about how the issuer wants? Right now all extra tokens are extracted at the time of finalization of the STO. **[Product Question]**